cmd/devenv/apps/run/run.go

```diff
@@ -93,6 +99,11 @@ func NewCmd(log logrus.FieldLogger) *cli.Command {
 				Aliases: []string{"t", "terminal"},
 				Usage:   "Open an interactive terminal to the dev container instead of running the application",
 			},
+			&cli.BoolFlag{
+				Name:    "skip-portforwarding",
+				Aliases: []string{"p"},
+				Usage:   "Skip forwarding ports. By default ports are forwarded to the dev container.",
```

Comment:
```suggestion
				Usage:   "Skip forwarding ports; useful when running multiple `devenv apps run` commands at once",
```
or maybe some such to make it more obvious why you'd want this